### ISSUE TITLE
Salesforce-specific violations are reported as composite values

### ISSUE THREAD

**Issue by SCWells72** [salesforce-ux/slds-linter#210]
I don't know if this is the right place to report this issue, but I'll start here and see where it goes. I've added the Salesforce-specific ESLint and Stylelint rules from SLDS Linter to configuration files for those respective static code analyzers, and overall it's working quite nicely, However, one thing I've noticed is that violations reported against Salesforce-specific rules are formatted as composite JSON values, e.g., the following from ESLint:

**@salesforce-ux/slds/enforce-bem-usage**
```json
{"message":"slds-button--brand has been retired. Update it to the new name slds-button_brand.","suggestions":["slds-button_brand"]}
```

and the following from Stylelint:

**slds/no-hardcoded-values-slds1**
```json
{"message":"Replace the rgb(221, 219, 218) static value with an SLDS 1 styling hook: \n1. --slds-g-color-palette-neutral-90\n2. --slds-g-color-neutral-base-90\n3. --slds-g-color-surface-3\n4. --slds-g-color-surface-container-3\n5. --slds-g-color-palette-neutral-80. (slds/no-hardcoded-values-slds1)","suggestions":["--slds-g-color-palette-neutral-90","--slds-g-color-neutral-base-90","--slds-g-color-surface-3","--slds-g-color-surface-container-3","--slds-g-color-palette-neutral-80"]}
```

However, violations against standard rules are reported as expected, e.g.:

**no-console**
```
Unexpected console statement.
```

and:

**color-no-invalid-hex**
```
Unexpected invalid hex color "#zzzzz" (color-no-invalid-hex)
```

Is that something that's configurable the respective config files? Apologies if this is obvious and I've just overlooked it. Also, please feel free to let me know if this question is better asked somewhere else such as wherever these Salesforce-specific rulesets are maintained.

**UPDATE:** Oddly, here's a Salesforce-specific Stylelint rule for which violations seem to be reported properly:

**slds/lwc-token-to-slds-hook**
```
The '--lwc-borderWidthThin' design token is deprecated. Replace it with the SLDS 2 '--slds-g-sizing-border-1' styling hook and set the fallback to '--lwc-borderWidthThin'. For more info, see Global Styling Hooks on lightningdesignsystem.com. (slds/lwc-token-to-slds-hook)
```

**Comment by sf-nithappu** [comment]
@SCWells72 We'll correct the console messages in upcoming release.

**Comment by sf-nithappu** [comment]
Hi @SCWells72 , We corrected our eslint and stylelint plugins to report violations in plain text format in latest release [0.5.0](https://github.com/salesforce-ux/slds-linter/releases/tag/0.5.0)
